Testscripts/Linux/ETHTOOL-MSG-LEVEL.sh

```diff
@@ -25,16 +25,14 @@ UtilsInit
 #######################################################################
 # Check if feature is supported by the kernel
 kernel_version=$(uname -r)
+
 # if the module is not builtin
-modules_path="/lib/modules/$kernel_version"
-if [ "$(grep -c netvsc < "$modules_path"/modules.builtin)" == 0 ]; then
+if kernel_module=$(modinfo -n hv_netvsc) ;then
```

Comment:
what is the output when hv_netvsc is built-in?

Reply:
It should print ```Module not found``` and should fall to else.

Reply:
Can we raise the quality of the logging? If you print out why the test failed with this reason, then testers do not need to RCA. They immediately know why it fails. It helps a lot.

Reply:
This case already have required logs.